src/Main.java

```diff
@@ -1,5 +1,119 @@
+
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Formatter;
+import java.util.logging.FileHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+import java.util.regex.Pattern;
+
 public class Main {
-    public static void main(String[] args) {
+    private static Logger logger = Logger.getLogger(Main.class.getName());
+
+    public static void main(String[] args) throws IOException {
+
+        // Setting logger and file handler to write files to web_traffic_report.txt
+        logger.setLevel(Level.INFO);
+        FileHandler handler = new FileHandler("src/web_traffic_report.txt");
+        SimpleFormatter formatter = new SimpleFormatter();
+        handler.setFormatter(formatter);
+        handler.setLevel(Level.INFO);
+        logger.addHandler(handler);
+
+
+        //Opening arraylist to store each request in the form of string in arraylist as object
+        ArrayList<String> request = new ArrayList<>();
+
+        //using BufferedReader class to read access.txt file
+        BufferedReader reader = new BufferedReader(new FileReader(new File("src/access.txt")));
+
+
+        //using while loop to assign each request to the arraylist named request;
+        int numberOfRequests = 0; // number of requests
+        String s;
+        while ((s = reader.readLine()) != null) {
+            request.add(s);
+            numberOfRequests++;
+        }
+        //creating REGEX for validating IP address
+        String zeroTo255 = "(\\d{1,2}|(0|1)\\" + "d{2}|2[0-4]\\d|25[0-5])";
+        String ipRegex = "\\[" + zeroTo255 + "\\." + zeroTo255 + "\\." + zeroTo255 + "\\." + zeroTo255 + "\\]";
+        //creating regex for date
+        String dateRegex = "\\[\\d{4}\\-\\d{2}\\-\\d{2}";
+        //creating regex for error code
+        String codeRegex="\\d{3}";
+
+
+
+
+        String[] request1 = request.get(0).split(" ");
+        String[] request2 = request.get(1).split(" ");
+        String[] request3 = request.get(2).split(" ");
+        String[] request4 = request.get(3).split(" ");
```

Comment:
How do you thinks it's possible to have more than 4 entities in a data file ? Why are writing statically splitting for only 4 log data